src/sql/src/session/vars/definitions.rs

```diff
@@ -1853,6 +1853,12 @@ feature_flags!(
         default: false,
         enable_for_item_parsing: true,
     },
+    {
+        name: partition_by,
```

Comment:
small nit, usually we prefix features with `enable_`
```suggestion
        name: enable_partition_by,
```